src/jvm/org/pantsbuild/example/proto/BUILD

```diff
@@ -0,0 +1,8 @@
+protobuf_sources(
+    grpc=True,
```

Comment:
TODO: This flag is not currently implemented for the JVM: https://github.com/pantsbuild/pants/issues/15691